### PR TITLE
Add has_no_root_node_jumps check

### PR DESCRIPTION
**Pull request by wizmer**
Check that the neurites have their first point not further than
`radius_multiplier * soma radius` from the soma center'''

### CODE REVIEW

**Comment by wizmer** [comment]
This is required for the validator web service I've been asked to build

File: neurom/check/neuron_checks.py [hunk 1]
```diff
@@ -223,6 +223,19 @@ def has_no_jumps(neuron, max_distance=30.0, axis='z'):
     return CheckResult(len(bad_ids) == 0, bad_ids)
 
 
+def has_no_root_node_jumps(neuron, radius_multiplier=2):
```
Comment (wizmer) [inline neurom/check/neuron_checks.py]: Not sure about the name of the function

Comment (wizmer) [reply neurom/check/neuron_checks.py]: Not sure about the name `radius_multiplier`

Comment (mgeplf) [reply neurom/check/neuron_checks.py]: I can't think of better names

**Pull request merged**